acceptance/helper.rb

```diff
@@ -335,7 +338,7 @@ def validate_package_version(host)
       expected_version = get_package_version(host)
 
       Beaker::Log.notify "Expecting package version: '#{expected_version}', actual version: '#{installed_version}'"
-      if installed_version != expected_version
+      if installed_version != expected_version and expected_version != 'latest'
         raise RuntimeError, "Installed version '#{installed_version}' did not match expected version '#{expected_version}'"
```

Comment:
Not sure I completely understand the context yet, but wondered if it might make sense for expected_version to be set to 'latest' when that's appropriate, instead of the extra guard.  Seemed like that might also improve the log message above.